Boolean_set_operations_2/doc/Boolean_set_operations_2/CGAL/draw_polygon_set_2.h

```diff
@@ -3,7 +3,8 @@ namespace CGAL {
 /*!
 \ingroup PkgDrawPolygonSet2
 
-opens a new window and draws `aps`, an instance of the `CGAL::Polygon_set_2` class. A call to this function is blocking, that is the program continues as soon as the user closes the window. This function requires CGAL_Qt5, and is only available if the flag CGAL_USE_BASIC_VIEWER is defined at compile time.
+opens a new window and draws `aps`, an instance of the `CGAL::Polygon_set_2` class. A call to this function is blocking, that is the program continues as soon as the user closes the window. This function requires `CGAL_Qt5`, and is only available if the macro `CGAL_USE_BASIC_VIEWER` is defined.
+Linking with the cmake target `CGAL::CGAL_Basic_viewer` will link with `CGAL_Qt5` add the definition.
```

Comment:
Suggestion for the last sentence:

Linking with the cmake target `CGAL::CGAL_Basic_viewer` will link with `CGAL_Qt5` and add the `CGAL_USE_BASIC_VIEWER` definition.